Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -77,7 +77,7 @@ class DropDownMenu_ItemInfo
       public IsDisabled: boolean,
       public readonly Href: Nullable<string>,
       public readonly Target: Nullable<string>,
-      public readonly FallbackNavigationUrl: string,
+      public readonly ClickHandler: Nullable<Function>,
```

Comment:
`Function` represents `any` function and will prevent type checking. Specifying the type here is better.

```suggestion
      public readonly ClickHandler: Nullable<() => boolean>,
```